### PR TITLE
Set searchbar label value and add prop to hide it

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
### Testing
View stories: home page, header, home hero, searchbar. Inspect the html and verify the nearby label is not empty, and is visually hidden.

### CODE REVIEW

File: src/library/components/Autocomplete/Autocomplete.styles.js [hunk 1]
```diff
@@ -1,79 +1,116 @@
-import styled from "styled-components";
```
Comment (DamienBurkeWN) [inline src/library/components/Autocomplete/Autocomplete.styles.js:1]: Mostly prettier...

File: src/library/components/Autocomplete/Autocomplete.styles.js [hunk 2]
```diff
@@ -1,79 +1,116 @@
...
+const hideLabel = props => {
+  if (props.labelHidden === true) {
+    return VisuallyHidden;
+  }
+}
+
 export const AutocompleteLabel = styled.label`
-    font-family: ${props => props.theme.theme_vars.fontstack};
+  font-family: ${props => props.theme.theme_vars.fontstack};
+  ${props => hideLabel}
```
Comment (DamienBurkeWN) [inline src/library/components/Autocomplete/Autocomplete.styles.js:38]: This is the real change, see hideLabel above

File: src/library/components/Autocomplete/Autocomplete.stories.tsx [hunk 3]
```diff
@@ -24,6 +24,9 @@ export default {
       labelText: {
         table: { category: "Input control" }
       },
+      labelHidden: {
```
Comment (DamienBurkeWN) [inline src/library/components/Autocomplete/Autocomplete.stories.tsx:27]: New control so the associated label can be hidden

File: src/library/components/Autocomplete/Autocomplete.tsx [hunk 4]
```diff
@@ -112,7 +113,7 @@ const Autocomplete: React.FunctionComponent<AutocompleteProps> = ({
           getRootProps
         }) => (
           <div>
-            <Styles.AutocompleteLabel {...getLabelProps()}>{labelText}</Styles.AutocompleteLabel>
+            <Styles.AutocompleteLabel {...getLabelProps()} labelHidden={labelHidden}>{labelText ? labelText : placeholder }</Styles.AutocompleteLabel>
```
Comment (DamienBurkeWN) [inline src/library/components/Autocomplete/Autocomplete.tsx:116]: Now uses placeholder text if no labeltext specified

**Comment by chrisrhymes** [comment]
Would something like this be useful for testing potential accessibility issues? https://github.com/nickcolley/jest-axe

File: src/library/components/Autocomplete/Autocomplete.tsx [hunk 5]
```diff
@@ -13,6 +13,7 @@ const Autocomplete: React.FunctionComponent<AutocompleteProps> = ({
   id = "autocomplete",
   name = "autocomplete",
   labelText,
+  labelHidden = false,
```
Comment (chrisrhymes) [inline src/library/components/Autocomplete/Autocomplete.tsx:16]: Is autocomplete used anywhere else right now?

Comment (DamienBurkeWN) [reply src/library/components/Autocomplete/Autocomplete.tsx:16]: Not yet

File: src/library/components/Autocomplete/Autocomplete.tsx (hunk 4 above)
Comment (chrisrhymes) [reply src/library/components/Autocomplete/Autocomplete.tsx:116]: The label could do with some styling when it is visible, such as some padding between it and the input.

**Comment by chrisrhymes** [comment]
Good opportunity to check that the label is set, but visible/ not visible with a test on the autocomplete component.

File: src/library/components/Autocomplete/Autocomplete.test.tsx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+} from './Autocomplete.storydata';
+import { west_theme } from '../../../themes/theme_generator';
+import { ThemeProvider } from 'styled-components';
+
+describe('Autocomplete component', () => {
+  it('should render a hidden label and empty input', () => {
+    const renderComponent = () =>
+      render(
+        <ThemeProvider theme={west_theme}>
+          <Autocomplete {...EmptyWithHiddenLabelData} />
```
Comment (chrisrhymes) [inline src/library/components/Autocomplete/Autocomplete.test.tsx:18]: Great idea to use the story data in the tests!

**Review by chrisrhymes — APPROVED** [review]

**Pull request merged**